setup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+import os
+from distutils.core import setup
+
+setup(
+    name='osmo_jupyter',
+    version='0.0.1',
+    author='Osmo Systems',
+    author_email='dev@osmobot.com',
+    description='Tools for accessing and processing OsmoBot data in an experimental context',

```

Comment:
If I could just think of a way to boil this description down into a three-word package name...

`experimental-data-processing-tools`?

Reply:
not `jupyter_python`? if you don't like that, plz engage in the unresolved conversation above